auth/api/iam/api.go

```diff
@@ -398,24 +464,71 @@ func (r Wrapper) RequestAccessToken(ctx context.Context, request RequestAccessTo
 	if !isWallet {
 		return nil, core.InvalidInputError("did not owned by this node")
 	}
-	if request.Body.UserID != nil && len(*request.Body.UserID) > 0 {
-		// forward to user flow
-		return r.requestUserAccessToken(ctx, *requestHolder, request)
+	return requestHolder, nil
+}
+
+func (r Wrapper) RequestUserAccessToken(ctx context.Context, request RequestUserAccessTokenRequestObject) (RequestUserAccessTokenResponseObject, error) {
+	if request.Body == nil {
+		// why did oapi-codegen generate a pointer for the body??
+		return nil, core.InvalidInputError("missing request body")
```

Comment:
Currently it can't, but better to prevent any panics in the future due to code changes by codegen.

Reply:
A panic only crashes the goroutine started for the particular HTTP request, so the Nuts node won't crash. In fact, better have it panic to signal (in the log) something is awfully wrong, instead of handling the request as if it were a user error (which it isn't). Because then, we don't notice it (just a "regular" HTTP 400).